### ISSUE TITLE
CommandBar accessibility: failed on rule aria-required-parent 

### ISSUE THREAD

**Issue by xugao** [microsoft/fluentui#15535]
### Environment Information

- **Package version(s)**: `@fluentui/react@latest`
- **Browser and OS versions**: (fill this out)

### Describe the issue:
Run accessibility insight Fast Pass on:
https://codepen.io/xugao/pen/dyXMbgQ
![image](https://user-images.githubusercontent.com/1207059/96198122-cbe6fd00-0f08-11eb-8e15-0527f3f820c4.png)
